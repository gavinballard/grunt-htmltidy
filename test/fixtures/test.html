<!doctype html>


<html>
  <body>
                <table><tr><td>some html </tr></table>
    <!-- Hello world -->

    <input    tyPE="submit" name="input"   />
  </body>





</html>





