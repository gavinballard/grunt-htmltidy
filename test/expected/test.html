<!DOCTYPE html>
<html>
<head>
<title></title>
</head>
<body>
<table>
<tr>
<td>some html</td>
</tr>
</table>
<!-- Hello world -->
<form><input type="submit" name="input"></form>
</body>
</html>